amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -74,11 +94,11 @@ describe('SearchPage', () => {
       props = setupResult.props;
       wrapper = setupResult.wrapper;
 
-      mockResourceType = ResourceType.user;
-      getSelectedTabByResourceTypeSpy = jest.spyOn(wrapper.instance(), 'getSelectedTabByResourceType').mockImplementation(() => {
-        return mockResourceType;
+      mockSatinitizedURLParams = { 'term': 'testName', ' index': 1, 'currentTab': 'table' };
```

Comment:
nit: this can also be renamed to `mockSatinitizedUrlParams`

Reply:
oops gotcha, also Satinized is definitely not the right word hahaha